torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -224,7 +224,12 @@ void runDiffGraphPasses(std::shared_ptr<Graph>& graph) {
     // TupleConstruct / TupleUnpack pairs can still be present at this point
     // and must be removed for fusion.
     LowerSimpleTuples(graph);
-    GRAPH_DUMP("After LowerSimpleTuples, before BatchMM", graph);
+    GRAPH_DUMP(
+        "After LowerSimpleTuples, before RemoveProfileNodesAndSpecializeTypes",
+        graph);
+    RemoveProfileNodesAndSpecializeTypes(graph);
```

Comment:
Can we call `EraseShapeInformation` if `!tensorExprFuserEnabled()`

Reply:
I moved here a call to `RemoveTensorTypeSpecializations` from the TE fuser - that should be equivalent, right?